src/FTPObject.jl

```diff
@@ -45,20 +45,11 @@ end
 @doc """
 Download the file "file_name" from FTP server and return IOStream.
 If "save_path" is not specified, contents are written to and returned as IOBuffer.
-"block" indicates if you want a blocking call, default is true
 """ ->
-function download(ftp::FTP, file_name::AbstractString, save_path::AbstractString=""; block::Bool=true)
-    if block
-        return download_internal(ftp.ctxt, file_name, save_path)
-    else
-        return remotecall(download_internal, myid(), ftp.ctxt, file_name, save_path)
-    end
-end
-
-function download_internal(ctxt::ConnContext, file_name::AbstractString, save_path::AbstractString)
+function download(ftp::FTP, file_name::AbstractString, save_path::AbstractString="")
     resp = nothing
     try
-        resp = ftp_get(ctxt, file_name, save_path)
+        resp = ftp_get(ftp.ctxt, file_name, save_path)
```

Comment:
I think we assumed that the body is what the user would care about.

Do you think users would like more info about the response, or should be just limit to the downlaod body and return nothing for upload?